### PR TITLE
rules

### PR DESCRIPTION
**Pull request by peterBrxwn**

### CODE REVIEW

**Comment by peterBrxwn** [comment]
[[RN] Disable typescript linter and enable only esLint linter (10.01)](https://trello.com/c/xbCIyccb/1053-rn-fix-some-linter-issues)

File: .eslintrc.json [hunk 1]
```diff
@@ -19,111 +18,186 @@
     "parser": "@typescript-eslint/parser",
     "plugins": ["react", "react-native", "@typescript-eslint"],
     "root": true,
+    "extends": ["airbnb/legacy"],
```
Comment (turtledreams) [inline .eslintrc.json:21]: at web sdk our reason to use this was we had to use esm5. here I think 6 would be fine too. rules can stay same but this part I think we can change.

**Review by turtledreams — CHANGES_REQUESTED** [review]

File: .eslintrc.json [hunk 2]
```diff
@@ -19,111 +18,186 @@
     "parser": "@typescript-eslint/parser",
```
Comment (arifBurakDemiray) [inline .eslintrc.json:18]: btw eslint is deprecating
https://eslint.org/blog/2023/10/deprecating-formatting-rules/

**Review by arifBurakDemiray — APPROVED** [review]
https://eslint.org/docs/latest/rules/no-class-assign
https://eslint.org/docs/latest/rules/no-cond-assign
https://eslint.org/docs/latest/rules/no-compare-neg-zero
https://eslint.org/docs/latest/rules/no-const-assign
maybe constants enforced

Also these can be added

**Pull request merged**